docs/qcqmc/full_workflow.ipynb

```diff
@@ -0,0 +1,363 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# End-to-End Example\n",
+    "\n",
+    "We learned in the [high-level](high-level.ipynb) notebook how to setup the quantum side of the calculation, which involves specifying a Hamiltonian, defining a trial wavefunction ansatz before performing shadow tomography to extract the AFQMC trial wavefunction. In this example we will repeat these steps again but now for the case of H4 which was studied in the [paper](https://arxiv.org/pdf/2106.16235.pdf). Then we will outline how to interface the output of the quantum half of the calculation with [ipie](https://github.com/JoonhoLee-Group/ipie/) to perform AFQMC with the quantum trial wavefunction."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Define an SCF Job\n",
+    "\n",
+    "We first setup a `PyscfHamiltonianParams` object which defines the SCF job we will run using [pyscf](https://github.com/pyscf/pyscf). Here we are simulation H4 in the (minimal) sto-3g basis."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from recirq.qcqmc.hamiltonian import PyscfHamiltonianParams, HamiltonianData\n",
+    "\n",
+    "pyscf_params = PyscfHamiltonianParams(\n",
+    "    name=\"TEST_square_H4\",\n",
+    "    n_orb=4,\n",
+    "    n_elec=4,\n",
+    "    geometry=((\"H\", (0, 0, 0)), (\"H\", (0, 0, 1.23)), (\"H\", (1.23, 0, 0)), (\"H\", (1.23, 0, 1.23))),\n",
+    "    basis=\"sto3g\",\n",
+    "    multiplicity=1,\n",
+    "    charge=0,\n",
+    "    save_chkfile=True,\n",
+    "    overwrite_chk_file=True,\n",
+    ")\n",
+    "pyscf_hamiltonian = HamiltonianData.build_hamiltonian_from_pyscf(pyscf_params)\n",
+    "chk_path = pyscf_params.base_path.with_suffix(\".chk\")"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Build Perfect Pairing Wavefunction\n",
+    "\n",
+    "Next we build a perfect pairing Hamiltonian based upon our `pyscf_params`"
```

Comment:
Is "perfect pairing Hamiltonian" defined elsewhere?  I personally don't know what that is.

Reply:
should be `perfect paring wavefunction`. Fixed.